### PR TITLE
New package: ICEMortgageTechnology,Inc.Encompass version 7.0.0.2

### PR DESCRIPTION
**Pull request by PckgrMiranda**
Checklist for Pull Requests
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.9 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.9.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/199085)

### CODE REVIEW

**Comment by PckgrMiranda** [comment]
This application did not have a silent install switch, however did have /exenoui which was used for the silent install switch.

**Comment by stephengillie** [comment]
Manual Validation ended with:
image

File: manifests/i/ICEMortgageTechnology,Inc/Encompass/7.0.0.2/ICEMortgageTechnology,Inc.Encompass.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.9.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: ICEMortgageTechnology,Inc.Encompass
+PackageVersion: 7.0.0.2
+InstallerType: exe
```
Comment (PckgrTom) [inline manifests/i/ICEMortgageTechnology,Inc/Encompass/7.0.0.2/ICEMortgageTechnology,Inc.Encompass.installer.yaml:6]:
```suggestion
InstallerType: exe
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Adobe.Acrobat.Reader.64-bit
```

File: manifests/i/ICEMortgageTechnology,Inc/Encompass/7.0.0.2/ICEMortgageTechnology,Inc.Encompass.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: ICEMortgageTechnology,Inc.Encompass
+PackageVersion: 7.0.0.2
+InstallerType: exe
+Dependencies:
+  PackageDependencies:
+    - PackageIdentifier: Adobe.Acrobat.Reader.64-bit
+InstallerSwitches:
+  Silent: /exenoui
```
Comment (PckgrTom) [inline manifests/i/ICEMortgageTechnology,Inc/Encompass/7.0.0.2/ICEMortgageTechnology,Inc.Encompass.installer.yaml:11]:
```suggestion
  Silent: /exenoui /qn /norestart
```

File: manifests/i/ICEMortgageTechnology,Inc/Encompass/7.0.0.2/ICEMortgageTechnology,Inc.Encompass.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.9.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.9.0.schema.json
+
+PackageIdentifier: ICEMortgageTechnology,Inc.Encompass
+PackageVersion: 7.0.0.2
+PackageLocale: en-US
+Publisher: Ellie Mae, Inc.
```
Comment (Trenly) [inline manifests/i/ICEMortgageTechnology,Inc/Encompass/7.0.0.2/ICEMortgageTechnology,Inc.Encompass.locale.en-US.yaml:7]:
```suggestion
Publisher: ICE Mortgage Technology, Inc.
```
This should match control panel

File: manifests/i/ICEMortgageTechnology,Inc/Encompass/7.0.0.2/ICEMortgageTechnology,Inc.Encompass.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,17 @@
+# Created using wingetcreate 1.9.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: ICEMortgageTechnology,Inc.Encompass
```
Comment (Trenly) [inline manifests/i/ICEMortgageTechnology,Inc/Encompass/7.0.0.2/ICEMortgageTechnology,Inc.Encompass.installer.yaml:4]:
```suggestion
PackageIdentifier: ICEMortgageTechnologyInc.Encompass
```
Commas are not preferred in package identifiers

File: manifests/i/ICEMortgageTechnology,Inc/Encompass/7.0.0.2/ICEMortgageTechnology,Inc.Encompass.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.9.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.9.0.schema.json
+
+PackageIdentifier: ICEMortgageTechnology,Inc.Encompass
```
Comment (Trenly) [inline manifests/i/ICEMortgageTechnology,Inc/Encompass/7.0.0.2/ICEMortgageTechnology,Inc.Encompass.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: ICEMortgageTechnologyInc.Encompass
```

File: manifests/i/ICEMortgageTechnology,Inc/Encompass/7.0.0.2/ICEMortgageTechnology,Inc.Encompass.yaml [hunk 6]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.9.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.9.0.schema.json
+
+PackageIdentifier: ICEMortgageTechnology,Inc.Encompass
```
Comment (Trenly) [inline manifests/i/ICEMortgageTechnology,Inc/Encompass/7.0.0.2/ICEMortgageTechnology,Inc.Encompass.yaml:4]:
```suggestion
PackageIdentifier: ICEMortgageTechnologyInc.Encompass
```

File: manifests/i/ICEMortgageTechnology,Inc/Encompass/7.0.0.2/ICEMortgageTechnology,Inc.Encompass.locale.en-US.yaml [hunk 7]
```diff
@@ -0,0 +1,13 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.9.0.schema.json
+
+PackageIdentifier: ICEMortgageTechnology,Inc.Encompass
+PackageVersion: 7.0.0.2
+PackageLocale: en-US
+Publisher: Ellie Mae, Inc.
+PackageName: Encompass360
+License: Proprietary
+Copyright: Copyright (C) 2024 ICE Mortgage Technology, Inc.
+ShortDescription: Encompass360 Installer
```
Comment (Trenly) [inline manifests/i/ICEMortgageTechnology,Inc/Encompass/7.0.0.2/ICEMortgageTechnology,Inc.Encompass.locale.en-US.yaml:11]: Please provide a better description of what Encompass is, does, or what it is used for

File: manifests/i/ICEMortgageTechnology,Inc/Encompass/7.0.0.2/ICEMortgageTechnology,Inc.Encompass.installer.yaml [hunk 8]
```diff
@@ -0,0 +1,17 @@
+# Created using wingetcreate 1.9.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: ICEMortgageTechnology,Inc.Encompass
+PackageVersion: 7.0.0.2
```
Comment (Trenly) [inline manifests/i/ICEMortgageTechnology,Inc/Encompass/7.0.0.2/ICEMortgageTechnology,Inc.Encompass.installer.yaml:5]:
```suggestion
PackageVersion: 2.0.1.0
```
This should match what is written to control panel

File: manifests/i/ICEMortgageTechnology,Inc/Encompass/7.0.0.2/ICEMortgageTechnology,Inc.Encompass.locale.en-US.yaml [hunk 9]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.9.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.9.0.schema.json
+
+PackageIdentifier: ICEMortgageTechnology,Inc.Encompass
+PackageVersion: 7.0.0.2
```
Comment (Trenly) [inline manifests/i/ICEMortgageTechnology,Inc/Encompass/7.0.0.2/ICEMortgageTechnology,Inc.Encompass.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 2.0.1.0
```

File: manifests/i/ICEMortgageTechnology,Inc/Encompass/7.0.0.2/ICEMortgageTechnology,Inc.Encompass.yaml [hunk 10]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.9.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.9.0.schema.json
+
+PackageIdentifier: ICEMortgageTechnology,Inc.Encompass
+PackageVersion: 7.0.0.2
```
Comment (Trenly) [inline manifests/i/ICEMortgageTechnology,Inc/Encompass/7.0.0.2/ICEMortgageTechnology,Inc.Encompass.yaml:5]:
```suggestion
PackageVersion: 2.0.1.0
```

**Review by Trenly — CHANGES_REQUESTED** [review]
After making the changes, the manifest folder name and the version folder will need to be updated

**Pull request closed without merging**